File: apps/auth-server/src/main/java/com/example/chat/auth/server/AuthServerApplication.java
package com.example.chat.auth.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

import com.example.chat.common.logging.annotation.EnableTracingLogging;

@EnableTracingLogging
@EnableDiscoveryClient
@SpringBootApplication(scanBasePackages = "com.example.chat")
public class AuthServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(AuthServerApplication.class, args);
    }
}

File: apps/auth-server/src/main/kotlin/com/example/chat/auth/server/AuthServerApplication.kt
package com.example.chat.auth.server

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@EnableDiscoveryClient
@SpringBootApplication(scanBasePackages = ["com.example.chat"])
class AuthServerApplication

fun main(args: Array<String>) {
    runApplication<AuthServerApplication>(*args)
}
